lms/templates/student_account/hinted_login.underscore

```diff
@@ -0,0 +1,24 @@
+<div class="hint-wrapper">
+    <div class="section-title lines">
+        <h2>
+            <span class="text"><%- gettext("Sign in") %></span>
+        </h2>
+    </div>
+
+    <p><%- _.sprintf( gettext("Would you like to sign in using your %(providerName)s credentials?"), { providerName: hintedProvider.name } ) %></p>
```

Comment:
Visual Feedback: This text is rendered currently in a serif typeface. Please make sure it renders in our Open Sans typeface. Our [typography Sass partial](https://github.com/edx/edx-platform/blob/master/lms%2Fstatic%2Fsass%2Felements%2F_typography.scss) could be your bestie for things like this - please extend a placeholder from it.